### ISSUE TITLE
Could not parse field WS2P

### ISSUE THREAD

**Issue by M5oul** [duniter/sakia#746]
```bash
DEBUG:node:refresh_block:[DJQjW] Changed block 60332 -> 60332
DEBUG:node:refresh_block:[DJQjW] Could not connect to any BMA endpoint
DEBUG:network:discovery_loop:New node found : HmH5b
ERROR:root:Task exception was never retrieved
future: <Task finished coro=<NetworkService.discovery_loop() done, defined at /home/moul/projects/sakia_git/src/sakia/services/network.py:169> exception=TypeError('Cannot convert ES_USER_API g1.data.le-sou.org 443 to endpoint',)>
Traceback (most recent call last):
  File "/home/moul/projects/sakia_git/src/sakia/services/network.py", line 185, in discovery_loop
    connector = NodeConnector.from_peer(self.currency, peer, self._app.parameters)
  File "/home/moul/projects/sakia_git/src/sakia/data/connectors/node.py", line 100, in from_peer
    node = Node(peer.currency, peer.pubkey, peer.endpoints, peer.blockUID, last_state_change=time.time())
  File "<attrs generated init 4e6ef8c0aaab861b7849411a9dfcc3e88901f518>", line 4, in __init__
    self.endpoints = __attr_convert_endpoints(endpoints)
  File "/home/moul/projects/sakia_git/src/sakia/data/entities/node.py", line 10, in _tuple_of_endpoints
    l = [endpoint(e) for e in value]
  File "/home/moul/projects/sakia_git/src/sakia/data/entities/node.py", line 10, in <listcomp>
    l = [endpoint(e) for e in value]
  File "/home/moul/.pyenv/versions/sakia-env/lib/python3.6/site-packages/duniterpy/documents/peer.py", line 106, in endpoint
    raise TypeError("Cannot convert {0} to endpoint".format(value))
TypeError: Cannot convert ES_USER_API g1.data.le-sou.org 443 to endpoint
DEBUG:node:refresh_peer_data:[D9D2z] Could not parse field WS2P
```

**Comment by Insoleet** [comment]
Should be fixed in 0.32.3 with latest duniter-python-api

**Comment by M5oul** [comment]
Seems fixed.
